src/components/atoms/SpacesDropdown/SpacesDropdown.tsx

```diff
@@ -49,7 +49,11 @@ export const SpacesDropdown: React.FC<SpacesDropdownProps> = ({
 
   // @debt: Probably need to omit returning playa from the useOwnedVenues as it's deprecated and
   // doesn't exist on SPACE_PORTALS_ICONS_MAPPING
-  const filteredSpaces = omit(spaces, VenueTemplate.playa);
+  const spacesWithoutPlaya = omit(spaces, VenueTemplate.playa);
+  const filteredSpaces = omitBy(
+    spacesWithoutPlaya,
+    (s) => s.template === VenueTemplate.posterpage
```

Comment:
What is the issue with `posterpage`? Should the explanation be added to the `@debt` above?

Reply:
I'll add a new comment. It's not debt

Reply:
Added in 95e328356